luxai2021/game/game.py

```diff
@@ -115,20 +119,34 @@ def reset(self, updates=None, increment_turn=False):
                 input_identifier = strings[0]
                 if input_identifier == INPUT_CONSTANTS.RESEARCH_POINTS:
                     team = int(strings[1])
```

Comment:
nit (cosmetic/readability): like on lines 140-143, use `research_points = int(string[2])` rather than repeating `int(string[2])` in the `if/else`

Reply:
pushed as you suggest